ci/jenkins/templates/utils/base.groovy.j2

```diff
@@ -91,41 +91,15 @@ docker_build = 'docker/build.sh'
 max_time = 180
 rebuild_docker_images = false
 
-// Filenames for stashing between build and test steps
-{% set tvm_runtime = ['build/libtvm_runtime.so', 'build/config.cmake'] %}
-{% set tvm_lib = ['build/libtvm.so'] + tvm_runtime %}
-{% set tvm_multilib = ['build/libtvm.so', 'build/libvta_fsim.so'] + tvm_runtime %}
-{% set tvm_multilib_tsim = ['build/libvta_tsim.so'] + tvm_multilib %}
-{% set microtvm_template_projects = ['build/microtvm_template_projects',] %}
-{% set hexagon_api = ['build/hexagon_api_output',] %}
 s3_prefix = "tvm-jenkins-artifacts-prod/tvm/${env.BRANCH_NAME}/${env.BUILD_NUMBER}"
 
-{% set aws_default_region = "us-west-2" %}
-{% set aws_ecr_url = "dkr.ecr." + aws_default_region + ".amazonaws.com" %}
-
 // General note: Jenkins has limits on the size of a method (or top level code)
 // that are pretty strict, so most usage of groovy methods in these templates
 // are purely to satisfy the JVM
-{% include "ci/jenkins/Prepare.groovy.j2" %}
-{% include "ci/jenkins/DockerBuild.groovy.j2" %}
-{% include "ci/jenkins/Lint.groovy.j2" %}
-{% include "ci/jenkins/Build.groovy.j2" %}
-{% include "ci/jenkins/Test.groovy.j2" %}
-{% include "ci/jenkins/Deploy.groovy.j2" %}
+{% include "utils/Prepare.groovy.j2" %}
+{% include "utils/Build.groovy.j2" %}
 
 
 cancel_previous_build()
 
 prepare()
```

Comment:
We could add a newline here

Reply:
The EOF-adjacent newlines get caught by https://github.com/apache/tvm/blob/main/tests/lint/whitespace.sh, so we can't really change them without adding an exception